### PR TITLE
Milestone1 basic project structure

### PR DESCRIPTION
**Pull request by DanielMitiku**
Finished basic project setup and ready to go on!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
```
Comment (jofer86) [inline src/App.js]: Here we should render just an empty div, with no content inside, also we need to get rid of all unnecessary files, that is the logo image, service workers, we should only be left off with a div with a class of App and nothing rendered inside.

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (jofer86) [inline src/App.test.js]: Let's remove this file, unless you are planning on doing testing

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (jofer86) [inline src/index.js]: let's remove this line as well

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (jofer86) [inline src/index.js]: let's remove this line, as we are not using service workers

File: src/logo.svg [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+        <circle cx="420.9" cy="296.5" r="45.7"/>
+        <path d="M520.5 78.1z"/>
+    </g>
+</svg>
```
Comment (jofer86) [inline src/logo.svg]: Let's remove this file as we are not using it

File: src/serviceWorker.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+  }
+}
+
+export function unregister() {
+  if ('serviceWorker' in navigator) {
+    navigator.serviceWorker.ready.then((registration) => {
+      registration.unregister();
+    });
+  }
+}
```
Comment (jofer86) [inline src/serviceWorker.js]: Lets remove this file as we are not using it also.

File: src/setupTests.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom/extend-expect';
```
Comment (jofer86) [inline src/setupTests.js]: This file needs to go too. We are not using it.

File: src/App.css [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (jofer86) [inline src/App.css]: Let's delete this file too, or the contents as the styling done here is not relevant for the project.

File: public/robots.txt [hunk 9]
```diff
@@ -0,0 +1,2 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
```
Comment (jofer86) [inline public/robots.txt]: Let's delete this file also.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## 👋 👋 Hello I'll be reviewing your PR. Let's take a look 👁 👁 👁  , shall we? 

- As the instructions for this project explains we need to remove all of the unnecessary files the CRA tools generate. 
image
- Below I commented on the files that need deletion, if you find extra files that are not going to be used please do delete them and ask for a new code review!
- And also let's make sure our app is deployed to Heroku as the instructions say, and provide the link for the live app, even though it might not contain much we need to make sure is up there.

**Comment by DanielMitiku** [comment]
I've removed unnecessary files. The app has been deployed to heroku. The link is found in the readme of the milestone1 branch.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next milestones. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**